pep-0646.rst

```diff
@@ -683,11 +687,17 @@ these type annotations.
 Backwards Compatibility
 =======================
 
-The grammar changes necessary for unpacking of type variable tuples using
-the star operator are almost all covered by PEP 637. The one exception
-is '``*args: *Ts``', which would require an additional change. (This
-change could be avoided by simply mandating this be written
-as ``Unpack[Ts]``, however.)
+In order to use the star operator for unpacking of ``TypeVarTuple`` instances,
+we would need to make two grammar changes:
+
+1. Star expressions must be made valid in index operations,
+   e.g. ``Tuple[*Ts]`` (but *not* in slice operations in general - e.g.
+   ``Tuple[*Ts:*Ts]`` is nonsensical and should remain invalid).
+2. We would need to make '``*args: *Ts``' valid.
```

Comment:
You'll also have to discuss what this actually does at runtime. Does it call `Ts.__iter__()` or do something else? What will show up in `__annotations__`?

Reply:
Ah, good point - I've added some details on this. (I haven't explicitly mentioned `__annotations__`, but I've mentioned what the `repr` of the object returned by `*Ts` is, which I hope should be sufficient.)